package/azure_li_services_spec_template

```diff
@@ -85,12 +85,6 @@ install -D -m 644 systemd/azure-li-storage.service \
 install -D -m 644 systemd/azure-li-system-setup.service \
     %{buildroot}%{_unitdir}/azure-li-system-setup.service
 
-%preun
-%service_del_preun azure-li-config-lookup.service azure-li-network.service azure-li-user.service azure-li-call.service azure-li-install.service azure-li-report.service azure-li-cleanup.service azure-li-machine-constraints.service azure-li-storage.service azure-li-system-setup.service
-
-%postun
-%service_del_postun azure-li-config-lookup.service azure-li-network.service azure-li-user.service azure-li-call.service azure-li-install.service azure-li-report.service azure-li-cleanup.service azure-li-machine-constraints.service azure-li-storage.service azure-li-system-setup.service
-
```

Comment:
We should put a comment in here to address the services or we'll get pushback again on package submission review. Maybe something along these lines:

While the package provides services all services are one-shot. Additionally upon initial run, i.e. on first boot the package itself gets removed vi the azure-li-cleanup implementation. This means there is no package update path as the package will be gone after initial boot. Therefore there is no need for the us of the "standard" systemd service handling macros.